src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -108,16 +109,100 @@ def insar_tops_burst(
     return Path('merged')
 
 
-# TODO add more parameters
-# TODO does the format need to be the same as for our INSAR_GAMMA products?
-# TODO unit test
-def make_parameter_file(out_path: Path, reference_scene: str, secondary_scene: str) -> None:
-    output = {
-        'reference_scene': reference_scene,
-        'secondary_scene': secondary_scene,
-    }
-    with out_path.open('w') as f:
-        json.dump(output, f)
+def make_parameter_file(
+    out_path: Path,
+    reference_scene: str,
+    secondary_scene: str,
+    swath_number: int,
+    reference_burst_number: int,
+    secondary_burst_number: int,
+    polarization: str = "VV",
+    azimuth_looks: int = 4,
+    range_looks: int = 20,
+    dem_name: str = "GLO_30",
+    dem_resolution: int = 30
+) -> None:
+    """Create a parameter file for the output product"""
+
+    SPEED_OF_LIGHT = 299792458.0
+    SPACECRAFT_HEIGHT = 693000.0
+    EARTH_RADIUS = 6337286.638938101
+
+    parser = etree.XMLParser(encoding='utf-8', recover=True)
+
+    ref_annotation_path = f'{reference_scene}.SAFE/annotation/'
+    ref_annotation = [file for file in os.listdir(ref_annotation_path) if os.path.isfile(ref_annotation_path + file)][0]
+
+    ref_manifest_xml = etree.parse(f'{reference_scene}.SAFE/manifest.safe', parser)
+    sec_manifest_xml = etree.parse(f'{secondary_scene}.SAFE/manifest.safe', parser)
+    ref_annotation_xml = etree.parse(f'{ref_annotation_path}{ref_annotation}', parser)
+    topsProc_xml = etree.parse('topsProc.xml', parser)
+    topsApp_xml = etree.parse('topsApp.xml', parser)
+
+    safe = '{http://www.esa.int/safe/sentinel-1.0}'
+    s1 = '{http://www.esa.int/safe/sentinel-1.0/sentinel-1}'
+    metadata_path = './/metadataObject[@ID="measurementOrbitReference"]//xmlData//'
+    orbit_number_query = metadata_path + safe + 'orbitNumber'
+    orbit_direction_query = metadata_path + safe + 'extension//' + s1 + 'pass'
+
+    ref_orbit_number = ref_manifest_xml.find(orbit_number_query).text
+    ref_orbit_direction = ref_manifest_xml.find(orbit_direction_query).text
+    sec_orbit_number = sec_manifest_xml.find(orbit_number_query).text
+    sec_orbit_direction = sec_manifest_xml.find(orbit_direction_query).text
+    ref_heading = float(ref_annotation_xml.find('.//platformHeading').text)
+    ref_time = ref_annotation_xml.find('.//productFirstLineUtcTime').text
+    slant_range_time = float(ref_annotation_xml.find('.//slantRangeTime').text)
+    range_sampling_rate = float(ref_annotation_xml.find('.//rangeSamplingRate').text)
+    number_samples = int(ref_annotation_xml.find('.//swathTiming/samplesPerBurst').text)
+    baseline_par = topsProc_xml.find('.//IW-2_Bpar_at_midrange_for_first_common_burst').text
```

Comment:
The IW-2 here changes based on which swath the burst is in. Perhaps switch to `f'.//IW-{swath_number}...'`.